queries.sql

```diff
@@ -42,3 +42,18 @@ SELECT AVG(weight_kg) FROM animals;
 SELECT neutered, COUNT(escape_attempts) FROM animals GROUP BY neutered;
 SELECT species, MIN(weight_kg), MAX(weight_kg) FROM animals GROUP BY species;
 SELECT species, AVG(escape_attempts) FROM animals  WHERE date_of_birth > '1990-01-01' AND date_of_birth <'2000-12-31'  GROUP BY species;
+
+SELECT animals.name FROM animals JOIN owners ON owner_id = owners.id WHERE owners.id = 4;
+SELECT animals.name FROM animals Join species ON species_id = species.id WHERE species.id = 1;
+SELECT owners.full_name, animals.name FROM owners RIGHT JOIN animals ON owner_id = owners.id;
```

Comment:
- [ ] Kindly note that this query failed to include an owner without any animal (Jodie Whittaker). Kindly change your `RIGHT JOIN` clause to `LEFT JOIN`. This will include all owners including ones without animals.